src/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/9f089b925d.js" crossorigin="anonymous"></script>
+    <title>To do list</title>
+</head>
+
+<body>
+    <div class="container">
+        <div class="todo">
+            <div class="caption">
+                <p>Today's To Do</p>
+                <i class="fas fa-sync"></i>
+            </div>
+            <div class="input-todo">
+                <form>
+                    <input id="newItem" class="new" placeholder="Add to your list..." type="text">
+                    <button type='submit' class='form-btn'><i
+                            class="fas fa-level-down-alt rotate add"></i></button>
+                </form>
+            </div>
+            <div class="conditional">
+                <ul class="List"></ul>
+                <div class="clear">
+                    <button type="button" class="link-button ">Clear all

```

Comment:
Thank you for your optional suggestion, but since I am using a button tag (not an input type="button" any native button ( of type button, submit or reset) are supported by all user agents and assistive technology according to [MDN web docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role)